uktrade/data-hub-frontend
# Enable the new interaction form

## Description of change

Replace the old interaction form with a new React-based.

## Test instructions

1. Go to [the new interaction form](https://data-hub-fro-enable-new-myxz1g.herokuapp.com/companies/5bb239e4-b227-4a84-bc2e-3420fc129447/interactions/create)
2. Confirm if the new form works the same as the old interaction form

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
